index.html

```diff
@@ -95,176 +103,31 @@ <h1 class="recent-works">My Recent Works</h1>
-          </div>
-        </section>
-      </div>
-
-      <div>
-        <section class="container3">
-          <div>
-            <img class="projectimg" alt="Project's webpage screenshot" src="img/work1.png" />
-          </div>
-          <div>
-            <h2>
-              Multi-Post Stories <br />
-              Gain+Glory
-            </h2>
-          </div>
-
-          <div class="btn-group">
-            <ul id="btn2">
-              <li><button class="ruby languages">Ruby on rails</button></li>
-              <li><button class="languages">CSS</button></li>
-              <li><button class="languages">JavaScript</button></li>
-              <li><button class="languages">HTML</button></li>
-            </ul>
-          </div>
-
-          <div class="divbtn-btn">
-            <button class="btn-btn">See project</button>
-          </div>
-        </section>
-      </div>
-
-      <div>
-        <section class="container3">
-          <div>
-            <img class="projectimg" alt="Project's webpage screenshot" src="img/work1.png" />
-          </div>
-
-          <div>
-            <h2>
-              Multi-Post Stories<br />
-              Gain+Glory
-            </h2>
-          </div>
-
-          <div class="item5 btn-group">
-            <ul id="btn3">
-              <li><button class="ruby languages">Ruby on rails</button></li>
-              <li><button class="languages">CSS</button></li>
-              <li><button class="languages">JavaScript</button></li>
-              <li><button class="languages">HTML</button></li>
-            </ul>
-          </div>
-
-          <div class="divbtn-btn">
-            <button class="btn-btn">See project</button>
-          </div>
-        </section>
-      </div>
-
-      <div>
-        <section class="container3">
-          <div>
-            <img class="projectimg" alt="Project's webpage screenshot" src="img/work1.png" />
-          </div>
-
-          <div>
-            <h2>
-              Multi-Post Stories<br />
-              Gain+Glory
-            </h2>
-          </div>
-
-          <div class="btn-group">
-            <ul id="btn4">
-              <li><button class="ruby languages">Ruby on rails</button></li>
-              <li><button class="languages">CSS</button></li>
-              <li><button class="languages">JavaScript</button></li>
-              <li><button class="languages">HTML</button></li>
-            </ul>
-          </div>
-
-          <div class="divbtn-btn">
-            <button class="btn-btn">See project</button>
-          </div>
-        </section>
-      </div>
-
-      <div>
-        <section class="container3">
-          <div>
-            <img class="projectimg" alt="Project's webpage screenshot" src="img/work1.png" />
-          </div>
-
-          <div>
-            <h2>
-              Multi-Post Stories<br />
-              Gain+Glory
-            </h2>
-          </div>
+ 
+     
+    
+    <div class="wrapper"  id="portfolio" >
```

Comment:
All your "Recent works" are missing in your website, you should display them dynamically using JavaScript, kindly, fix it.

![image](https://user-images.githubusercontent.com/60164239/131812653-e7001150-1600-4943-91b6-b1db88fbc615.png)